### PR TITLE
remove item labels

### PR DESCRIPTION
**Pull request by mekolat**
## requires https://github.com/themanaworld/tmwa-server-data/pull/340

### CODE REVIEW

**Comment by mekolat** [comment]
@wushin I'm gonna cry
that bug is killing me; I don't understand what is wrong

**Comment by mekolat** [comment]
@wushin should square brackets be removed from getitemlink? IMO they should be added directly in the scripts if needed but not enforced

**Comment by nashley** [comment]
@mekolat IMO, the brackets should be removed entirely.

**Comment by mekolat** [comment]
That's my opinion too

**Comment by wushin** [comment]
I believe manaplus does some client side manipulation as well.

**Comment by wushin** [comment]
if you want to change the item link style use a separate PR we already have so much merged conflicts are happening.

File: src/ast/item_test.cpp [hunk 1]
```diff
@@ -89,9 +89,9 @@ namespace item
         {
             //        1         2         3         4         5
             //2345678901234567890123456789012345678901234567890123456789
-            "1,abc ,  def,3,4,5,6,7,8,9,10,xx,2,16,12,13,11, {end;}, {}"_s,
-            "1,abc ,  def,3,4,5,6,7,8,9,10,xx,2,16,12,13,11, {end;}, {}\n"_s,
-            "1,abc ,  def,3,4,5,6,7,8,9,10,xx,2,16,12,13,11, {end;}, {}\nabc"_s,
```
Comment (wushin) [inline src/ast/item_test.cpp]: can we fix this strange formatting left?

Comment (mekolat) [inline src/ast/item_test.cpp]: @wushin it is intended to have spaces: https://github.com/themanaworld/tmwa/pull/69/files#diff-b80c536e3ed1705979078a73d434113bR110

Comment (wushin) [inline src/ast/item_test.cpp]: ok.

**Pull request merged**